frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/clearingStatus.jspf

```diff
@@ -66,24 +67,26 @@
             });
         });
 
-        function renderActions(releaseId) {
-            return "<img src='<%=request.getContextPath()%>/images/fossology-logo-24.gif' " +
-                " class='action send-to-fossology' data-release-id='" + releaseId + "'" +
-                "  alt='SelectClearing' title='send to Fossology'>";
-        }
-
         function createClearingTable() {
             var result = [];
 
             <core_rt:forEach items="${releasesAndProjects}" var="releasesClearingStatusData">
+            var sendToFossology = "<img src='<%=request.getContextPath()%>/images/fossology-logo-24.gif' " +
+                                       " class='action send-to-fossology' data-release-id='" + releaseId + "'" +
+                                       "  alt='SelectClearing' title='send to Fossology'>";
+            var editRelease = "<sw360:DisplayReleaseLink releaseId='${releasesClearingStatusData.release.id}' showName='false' page='<%=PortletReleasePage.EDIT%>'>"+
```

Comment:
```suggestion
            editRelease = "<sw360:DisplayReleaseLink releaseId='${releasesClearingStatusData.release.id}' showName='false' page='<%=PortletReleasePage.EDIT%>'>"+
```

Reply:
I have pushed the requested changes.
Thanks